ros2/r2b2/ubuntu/xenial/images.yaml.em

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+# ROS2 Dockerfile database
+---
+images:
+    @(ros2distro_name)-core:
+        base_image: @(os_name):@(os_code_name)
+        maintainer_name: @(maintainer_name)
+        template_name: docker_images/create_ros2_core_image.Dockerfile.em
+        entrypoint_name: docker_images/ros2_entrypoint.sh
+        template_packages:
+            - docker_templates
+        packages:
+            - python3-pip
+        pip3_install:
+            - argcomplete
+            - flake8
+            - flake8-import-order
+        ros2_packages:
+            - demo-nodes-cpp
+            - demo-nodes-py
+            - ros2run
+            - sros2
+    @(ros2distro_name)-bridge:
+        base_image: osrf/@(user_name):@(ros2distro_name)-core
+        maintainer_name: @(maintainer_name)
+        template_name: docker_images/create_ros2_image.Dockerfile.em
+        entrypoint_name: docker_images/ros2_entrypoint.sh
+        template_packages:
+            - docker_templates
+        packages:
+            - python-rospkg
+        ros_packages:
+            - catkin
+            - common-msgs
+            - rosbash
+            - roscpp
+            - roscpp-tutorials
+            - roslaunch
+            - rosmsg
+            - rospy-tutorials
+            - tf2-msgs
+            - .*-*msg
+        ros2_packages:
+            - ros1-bridge
```

Comment:
In the end I think that a Dockerfile installing only `ros-r2b2-ros1-bridge` will match the requirements here given that it already provides all the packages needed to "communicate joystick commands over the bridge as well as all messages to visualize a robot in rviz."
For convenience we could throw in `ros2run`.

> as it was not entirely obvious to me which ros1 distro pairs with this ros2 beta

Please consider contributing to the [wiki installation page](https://github.com/ros2/ros2/wiki/Linux-Install-Debians) to make it more obvious to users, improvements are always appreciated!